src/pymor/operators/constructions.py

```diff
@@ -155,6 +155,18 @@ def jacobian(self, U, mu=None):
         else:
             return jac
 
+    def mu_derivative(self, component, coordinates=None):
+        for op in self.operators:
+            if op.parametric:
+                raise NotImplemented
+        derivative_coefficients = []
+        for coef in self.coefficients:
+            try:
+                derivative_coefficients.append(coef.partial_derivative(component, coordinates))
+            except:
+                derivative_coefficients.append(0.)
+        return LincombOperator(self.operators, derivative_coefficients)
```

Comment:
`return self.with_(coefficients=derivative_coefficients, name=self.name + '_d_mu')` might be a bit more idiomatic and it also preserves `solver_options`.